events-api.go

```diff
@@ -12,12 +12,14 @@ import (
 
 	"github.com/Kable-io/kable-go/internal/openapi"
 	"github.com/ehsaniara/gointerlock"
+	"github.com/google/uuid"
 )
 
 type Event struct {
-	ClientId  string                 `json:"clientId"`
-	Data      map[string]interface{} `json:"data,omitempty"`
-	Timestamp *time.Time             `json:"timestamp,omitempty"`
+	ClientId      string                 `json:"clientId"`
+	Data          map[string]interface{} `json:"data,omitempty"`
+	TransactionId string                 `json:"transactionId,omitempty"`
```

Comment:
`Timestamp` is currently "optional" I believe by way of the `*`. Is it possible to handle `TransactionId` in the same way as `Timestamp`?

Reply:
I actually believe this is because string is a primitive and Time is not. In this case, we need to deference the Time type here.